docs/wasm/api.md

```diff
@@ -110,13 +110,43 @@ const transaction = {
     "params": ""
 };
 
-const cbor_transaction =  signer_wasm.transaction_serialize(JSON.stringify(transaction));
+const cbor_transaction =  signer_wasm.transactionSerialize(JSON.stringify(transaction));
 
 //
 console.log(cbor_transaction);
 ```
 
-## transaction_parse
+## transactionSerializeRaw
+
+Serialize a transaction and return a CBOR buffer.
+
+Arguments :
+* **transaction**: a filecoin transaction;
+
+```javascript
+const signer_wasm = require('@zondax/filecoin-signer');
+// or for browser
+// import * as signer_wasm from "@zondax/filecoin-signer";
+
+const transaction = {
+    "to": "t17uoq6tp427uzv7fztkbsnn64iwotfrristwpryy",
+    "from": "t1b4zd6ryj5dsnwda5jtjxj6ptkia5e35s52ox7ka",
+    "nonce": 1,
+    "value": "100000",
+    "gasprice": "2500",
+    "gaslimit": "25000",
+    "method": 0,
+    "params": ""
+};
+
+const cbor_transaction =  signer_wasm.transactionSerializeRaw(JSON.stringify(transaction));
```

Comment:
we probably don't need the stringify here. I think I improve this so plain dictionaries were also working.
We can check later